src/TMLeague/wwwroot/data/leagues/ptl/ptl.json

```diff
@@ -4,13 +4,23 @@
   "BackgroundImage": "data/leagues/ptl/ptl.jpg",
   "Rules": "https://www.thronemaster.net/?goto=community&sub=forum&fid=15&tid=76410",
   "Discord": null,
-  "Seasons": [ "s8" ],
+  "Seasons": [ "s1", "s2", "s3", "s4", "s5", "s6", "s7", "s8" ],
   "TrainingSeasons": [],
   "MainDivisions": [
     {
       "Id": "d1",
       "Name": "Division 1",
       "Relegations": 0
+    },
+	{
```

Comment:
Are you sure that `Qualifier 1` and `Qualifier 2` should be main divisions? Being main division means that stats are calculated for that divisions and they are included in Hall of Fame, but nothing more. I advice to restrict it only to really elite divisions. ;)

Reply:
Qualifiers merged into one "big" division